### DISCUSSION TITLE
Correct use of GeometryDecorator

### DISCUSSION THREAD

**Discussion by candcconsulting** [iTwin/itwinjs-core#6606] [Q&A]
I am using a GeometryDecorator which is being loaded with ~300 shapes
The createGraphics is being called as expected and the builder
at this point here
   const graphic = builder.finish();
has 644 pieces of geometry in the accum property
however, after this line graphics is an empty array
I have a sandbox here
**im me for the sandbox**
with the generated data preloaded ...

**Comment by pmconne** [comment]
Sandboxes are publicly viewable by anyone. If that's a problem please delete the sandbox; otherwise, please provide the link here.

Please describe how to reproduce the problem using your sandbox.
